src/Workleap.Extensions.OpenAPI/OpenApiServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Workleap.Extensions.OpenAPI.Builder;
+
+namespace Workleap.Extensions.OpenAPI;
+
+/// <summary>
+/// Provides extension methods to the <see cref="IServiceCollection"/> for configuring OpenAPI/Swagger services.
+/// </summary>
+public static class OpenApiServiceCollectionExtensions
+{
+    /// <summary>
+    /// Configures OpenAPI/Swagger document generation and SwaggerUI.
+    /// </summary>
+    public static OpenApiBuilder AddOpenApi(this IServiceCollection services)
```

Comment:
I would change this method name to something like `AddOpenAPIHelper` or `AddOpenAPIGen`?

also nit: `AddOpenAPI`